packages/foreman/foreman-installer/foreman-installer.spec

```diff
@@ -30,13 +30,29 @@ BuildRequires: puppet-agent-puppet-strings < 2
 %description
 Complete installer for The Foreman life-cycle management system based on Puppet.
 
+%package katello
+Summary: Katello installer bits
+Group: Applications/System
+Obsoletes: katello-installer-base < 3.11
+Conflicts: katello-installer-base < 3.11
+Provides: katello-installer-base < 3.11
+
+Requires: %{name} = %{version}-%{release}
+Requires: openssl
+Requires: katello-selinux
+Requires: katello-certs-tools
+Requires: katello-service >= 3.0.0
```

Comment:
Something to put on our list, this was "deprecated" in favor of foreman-maintain but I do see it still referenced in a few hooks.

Reply:
I also realized that when I saw this packaging side and agree it'd be good to avoid the deprecated dependency here.